class ObjectAlreadyExistsException(Exception):
    pass


class ObjectNotFoundException(Exception):
    pass
